src/algebra/order/with_zero.lean

```diff
@@ -171,6 +171,24 @@ begin
   simpa [inv_mul_cancel_left₀ ha, inv_mul_cancel_left₀ (ne_of_gt hc)] using this,
 end
 
+lemma with_zero.div_le_div_right (hc : c ≠ 0) : a/c ≤ b/c ↔ a ≤ b :=
+begin
+  rw [div_eq_mul_inv _ _, div_eq_mul_inv _ _],
+  split; intro hab,
+  { rw [← mul_one a, ← mul_one b, ← inv_mul_cancel hc, ← mul_assoc, ← mul_assoc],
+    exact mul_le_mul_right' hab _, },
+  { exact mul_le_mul_right' hab _, }
+end
+
+lemma with_zero.div_le_iff (hb : b ≠ 0) : a/b ≤ c ↔ a ≤ c*b :=
+begin
+  conv_lhs {rw ← mul_one c, rw ← div_self hb, rw ← mul_div_assoc},
+  exact with_zero.div_le_div_right hb,
+end
```

Comment:
You can golf these by splitting out the `div` part from the `mul_inv` part:
```suggestion
lemma mul_le_mul_right₀ (hc : c ≠ 0) : a * c ≤ b * c ↔ a ≤ b :=
⟨le_of_le_mul_right hc, λ hab, mul_le_mul_right' hab _⟩

lemma div_le_div_right₀ (hc : c ≠ 0) : a/c ≤ b/c ↔ a ≤ b :=
by rw [div_eq_mul_inv, div_eq_mul_inv, mul_le_mul_right₀ (inv_ne_zero hc)]

lemma mul_inv_le_iff₀ (hb : b ≠ 0) : a * b⁻¹ ≤ c ↔ a ≤ c * b :=
⟨λ h, inv_inv₀ b ▸ le_mul_inv_of_mul_le (inv_ne_zero hb) h, mul_inv_le_of_le_mul hb⟩

lemma div_le_iff₀ (hb : b ≠ 0) : a/b ≤ c ↔ a ≤ c*b :=
by rw [div_eq_mul_inv, mul_inv_le_iff₀ hb]
```

Reply:
I think `mul_inv_le_iff₀` probably belongs right after `mul_inv_le_of_le_mul`

Reply:
Good point! I'll change it now.